app/helpers/application_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module ApplicationHelper
   def link_to_switch_locale
-    if params[:locale] == 'en-US'
+    if I18n.locale == :'en-US'
```

Comment:
Yeah this was curious to me to (since @eitoball mentioned it in the other Pr)

Looks like it’s a constant (presumably the system locale) rather than the user selected locale. Is that right?

Reply:
Seems to work though. https://dev.safecast.org/en-US

The locale selector in this site has always confused me really. Seems like it should be header or session based rather than the route.

Reply:
You just basically compare what is the current locale. The problem was that each time user lands to root URL, `params[: locale]` will be `nil` resulting in showing English while default in settings already was set to English https://github.com/Safecast/safecastapi/blob/master/config/application.rb#L33

Reply:
We had a discussion on a call a while back about inconsistent handling of locale, it really needs more improvement at some point in the future.

That being said, even though this looks like a global constant to me (?), `I18N.locale` is set here: https://github.com/Safecast/safecastapi/blob/master/app/controllers/application_controller.rb#L59 which is also what's recommended in https://guides.rubyonrails.org/v4.2/i18n.html